docker-alpine/README.md

```diff
@@ -0,0 +1,58 @@
+# Docker for Alpine
+
+## Prerequisites
+1. Cross-compile with musl. This is an example how to cross-compile for x86_64 with musl.
+
+```shell
+cargo install cross
+cross build --release --target=x86_64-unknown-linux-musl
+```
```

Comment:
This step could be done by docker itself if we only add few of build command in the dockerfile, with multistage docker build you can either use docker to build for you or skip to copy and run stage.